coffee-frontend/package.json

```diff
@@ -4,10 +4,13 @@
   "private": true,
   "proxy": "http://localhost:8080",
   "dependencies": {
+    "@material-ui/core": "^4.9.10",
+    "@material-ui/icons": "^4.9.1",
```

Comment:
Shuldn't it be `4.9.10`?

Reply:
Nope, icons latest package is 4.9.1 for some reason